modules/mono/interop_types.h

```diff
@@ -40,7 +40,7 @@ extern "C" {
 #include <stdbool.h>
 #include <stdint.h>
 
-// This is taken from the old GDNative, which was removed.
+// This is taken from the old GDExtension, which was removed.
```

Comment:
The comment says this was taken from the old GDNative headers, which were removed. Renaming this here would cause confusion.